app/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "123456789012",
+    "firebase_url": "https://oppia-android-51g85.firebaseio.com",
+    "project_id": "oppia-android-51g85",
+    "storage_bucket": "oppia-android-51g85.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:123456789012:android:d544ded26ae57f7fe6e25b",
+        "android_client_info": {
+          "package_name": "org.oppia.app"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "123456789012-4unv88bb0rj9c4ki29hanl213qg6ter8.apps.googleusercontent.com",
```

Comment:
Ditto here--can we clearly indicate that this is a fake string?

Reply:
Done